src/role_sourcer.js

```diff
@@ -14,25 +14,24 @@
 roles.sourcer = {};
 
 roles.sourcer.settings = {
-  param: ['controller.level'],
+  param: ['energyCapacityAvailable'],
   prefixString: {
-    1: 'MW',
-    3: 'MWC'
+    150: 'MW',
+    600: 'MWC'
   },
   layoutString: {
-    2: 'W',
-    3: 'MW',
-    5: 'MW'
+    0: 'W',
```

Comment:
Ok, I think it doesn't matter ... somehow.

True 300 energy is the default minimum, but it wouldn't matter to use 0, correct?
Usually, I see private server where some constants are increased, but you could also decrease it, let's say spawn capacity 200, which would break it ....